driver/escaping/escape_sequences.cpp

```diff
@@ -39,13 +39,17 @@ const std::map<const Token::Type, const std::string> function_map{
     {Token::ABS, "abs"},
     {Token::CONCAT, "concat"},
     {Token::CURDATE, "today"},
+    {Token::CURRENT_DATE, "today"},
     {Token::TIMESTAMPDIFF, "dateDiff"},
+    {Token::SQL_TSI_QUARTER, "toQuarter"},
+    {Token::DAYOFWEEK, "toDayOfWeek"},
```

Comment:
Tableau's usage of DAYOFWEEK function has Sunday=1 (https://community.tableau.com/thread/204116), while clickhouse toDayOfWeek starts with Monday=1, so trivial translation will not work correctly here.

Reply:
ok, fixed.